hr_timesheet_name_customer/tests/test_name_customer.py

```diff
@@ -0,0 +1,112 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestCommonNameCustomer(TransactionCase):
```

Comment:
I think this class you can move to new common.py file and them make import to current file.